crates/sui-indexer-alt-framework/src/lib.rs

```diff
@@ -282,20 +238,14 @@ impl<S: Store> Indexer<S> {
             );
         }
 
-        // If an override has been provided, start ingestion from there, otherwise start ingestion
-        // from just after the lowest committer watermark across all enabled pipelines.
-        let first_checkpoint = self
-            .first_checkpoint
-            .unwrap_or(self.first_checkpoint_from_watermark);
-
         let last_checkpoint = self.last_checkpoint.unwrap_or(u64::MAX);
 
-        info!(first_checkpoint, last_checkpoint = ?self.last_checkpoint, "Ingestion range");
+        info!(self.first_checkpoint_from_watermark, last_checkpoint = ?self.last_checkpoint, "Ingestion range");
```

Comment:
I understand we probably call this `first_checkpoint_from_watermark` to distinguish from `first_checkpoint` which is a user-provided arg, but this name seems a bit imprecise because this value might not come from watermarks. It's really just the minimum of `next_checkpoint` across all pipelines. Maybe call it something like `min_next_checkpoint`?